spec/facter/resolvers/networking_linux_spec.rb

```diff
@@ -100,6 +107,10 @@ def ifaddr_obj(name, ip, mac, netmask, ipv4_type)
       expect(networking_linux.resolve(:primary_interface)).to eq('ens160')
     end
 
+    it 'returns the primary6 interface' do
```

Comment:
It seems like the `returns the primary(6) interface` tests up here are duplicates of the ones in `context 'when asking for primary interface' do`. :question:

Reply:
Yes, you are correct. In `context 'when asking for primary interface'` at line 467 we test scenarios when the different ways of extracting the primary interface fail. 
So, here you should add a test for what should happen when `ip -json -6 route show default` fails to retrieve the primary(6) interface.